notesapi/v1/tests/test_models.py

```diff
@@ -20,6 +20,7 @@ def setUp(self):
                     "endOffset": 10,
                 }
             ],
+            "tags": [u"apple", u"pear"],
```

Comment:
There is another test point that creates a note and verifies all the fields work. I didn't have to change that test method at all-- it just worked.